Tryouts/Prototypes/Shell/Shell.csproj

```diff
@@ -10,9 +10,14 @@
 
   <ItemGroup>
     <None Remove="MainWindow.xaml" />
+    <None Remove="Node-launcher\Lib\bin\cli.js" />
     <None Remove="WebWindow.xaml" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Include="Node-launcher\Lib\cli\cli.js" />
```

Comment:
You don't want to compile JS files in a csproj. Basically all the JS projects should be moved outside of this Shell project.

Reply:
Visual studio picked this up, I'm happy to remove it.

Reply:
All the JS projects should be removed from the .NET solution folder, but I can do this as part of the repo cleanup once we stop submitting new PRs.